data/tlg0559/tlg0559.tlg016.xml

```diff
@@ -0,0 +1,1664 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader>
+<fileDesc>
+<titleStmt>
+<title xml:lang="lat">Heronis Alexandrini geometricorum et stereometricorum reliquiae</title>
```

Comment:
The name of the author should not be in the text of the `<title>` attribute. Only the title of the work itself. And according to the TLG, the title of the work should be "Liber geeponicus".